application.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
```

Comment:
Out of curiosity why did you create a new application.rb file and not use the one in the config folder?

Reply:
Didn't actually mean to do that, git wouldn't commit any of my changes in bash so I tried the upload tool on the site. Sorry it's my first time using git

Reply:
I can't overwrite application.rb so I'll just close this and start a new pull request